### ISSUE TITLE
Potential developments

### ISSUE THREAD

**Issue by shivamsaboo17** [YashAndonia/Mouse_concept#1]
I think you left many cases where you won't be able to detect sliding finger action. Consider a case when user's thumb isn't completely horizontal to cover entire sensor region or is maybe too short. In this case you may encounter sequences like due to high values from uncovered sensors:
o _ _
_ o _
_ _ o 
i.e 011 101 110

**Comment by YashAndonia** [comment]
Yes, will consider it. Or perhaps such cases should be ignored completely?

**Comment by YashAndonia** [comment]
not sure if _0_ can happen. Sensors will be quite close. This system considers a sliding action from 0__ to 00_ to 000 and also from __0 to _00 to 000 as well for the other action. So it should work like that too

**Comment by shivamsaboo17** [comment]
Well, they can't be ignored for sure as they are valid cases.
Keeping sensors close will do the job but again how close? It shouldn't happen like the user has to search for the sensor zone to do anything. If you place the sensor zone strategically like the lock button on our phones, it would make life simpler.
Rather than hard-coding all the cases you may as well use more number of sensors and make a generalized prediction.
You may find this helpful https://www.amazon.in/Generic-APDS-9960-Gesture-Detection-Breakout/dp/B01KT7YN7Y?tag=googinhydr18418-21

**Comment by YashAndonia** [comment]
Will look into it, love the suggestion
